apps/users/tests/test_forms.py

```diff
@@ -349,9 +349,9 @@ def test_redirect_after_login(self):
         url = urlparams(self._get_login_url(), to="/en-US/firefox/about")
         r = self.client.post(url, {'username': 'jbalogh@mozilla.com',
                                    'password': 'password'}, follow=True)
-        self.assertRedirects(r, '/en-US/about')
+        self.assert3xx(r, '/en-US/about')
 
-        # Test a valid domain.  Note that assertRedirects doesn't work on
+        # Test a valid domain.  Note that assert3xx doesn't work on
```

Comment:
I think this comment is now irrelevant. `assert3xx` doesn't do a `client.get`, and so doesn't even have this limitation.